test/test_integration.rb

```diff
@@ -69,16 +69,8 @@ def wait_booted
   def restart_server(connection)
     signal :USR2
 
-    sleep 1
-
     connection.write "GET / HTTP/1.1\r\n\r\n" # trigger it to start by sending a new request
 
-    assert_raises Errno::ECONNRESET do
```

Comment:
the whole point of the soft restart is that the connection is not interrupted, so I don't think this made much sense ...